cime_config/config_grids.xml

```diff
@@ -1678,6 +1688,16 @@
       <mask>oEC60to30v3</mask>
     </model_grid>
 
+    <model_grid alias="ne30pg2_r05_EC30to60E2r2_gis1to10" compset="_MALI">
```

Comment:
see note below - do we really want to add both this and the R2 version?

Reply:
I will leave this question open for now (see initial response to this above and comment further there).